projects/packages/sync/src/replicastore/class-table-checksum-user.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * Table Checksums Class.
+ *
+ * @package automattic/jetpack-sync
+ */
+
+namespace Automattic\Jetpack\Sync\Replicastore;
+
+/**
+ * Class to handle Table Checksums for the User Meta table.
+ */
+class Table_Checksum_User extends Table_Checksum {
```

Comment:
This is a nitpick, but since this handles checksums for the `wp_users` table, should this class be named `Table_Checksum_Users`?